test/NuGet.Clients.Tests/NuGet.VisualStudio.Implementation.Test/Extensibility/VsPathContextProviderTests.cs

```diff
@@ -53,7 +51,8 @@ public void GetSolutionPathContext_WithConfiguredUserPackageFolder()
                 settings,
                 Mock.Of<IVsSolutionManager>(),
                 Mock.Of<ILogger>(),
-                getLockFileOrNullAsync: null);
+                getLockFileOrNullAsync: null,
+                _telemetryProvider.Object);
```

Comment:
Same as my other comment, should all these Tests check for an exception (or any output) existing/not-existing in telemetry?

Reply:
the mock object would throw an exception if any method or property was called, since it's mocked in strict mode.